server/tests/api/auth.test.ts

```diff
@@ -0,0 +1,203 @@
+import chai from "chai";
+import chaiHttp from "chai-http";
+import "mocha";
+import User from "../../src/models/User/User";
+import IUser from "../../src/models/User/IUser";
+import app from "../../src/app";
+import { generateOtpHash } from "../../src/utility/otp";
+
+chai.use(chaiHttp);
+const expect = chai.expect;
+
+describe("Auth tests", () => {
+
+    afterEach((done) => {
+        User.deleteMany().then(() => done());
+    })
+
+    describe("Bypassing VerifyOTP middleware", () => {
+        const email = "test-email@test.in";
+
+        let otp: number, hash: string;
+        beforeEach(() => {
+            [otp, hash] = generateOtpHash(email);
+        })
+        afterEach((done: Mocha.Done) => {
+            User.remove().then(() => done());
```

Comment:
.remove() has been deprecated, can we replace this by deleteOne() or deleteMany() perhaps?